### PR TITLE
Feature: Add Continuous Integration

### PR DESCRIPTION
**Pull request by vuquangit**

### CODE REVIEW

File: vite.config.ts [hunk 1]
```diff
@@ -5,4 +5,9 @@ import tsconfigPaths from 'vite-tsconfig-paths'
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react(), tsconfigPaths()],
+  resolve: {
```
Comment (trinhdinhhiep) [inline vite.config.ts]: why dont use vite-tsconfig-paths(tsconfigPaths) to path alias?

File: src/styles/index.css [hunk 2]
```diff
@@ -0,0 +1,14 @@
+@import './reset';
```
Comment (trinhdinhhiep) [inline src/styles/index.css]: Tuệ suggests using normalizecss library, is reset css necessary?

**Review by tuev — APPROVED** [review]
LGTM

**Review by trinhdinhhiep — APPROVED** [review]

**Comment by tuev** [comment]
@CodiumAI-Agent /describe

**Comment by tuev** [comment]
@CodiumAI-Agent /review

**Comment by tuev** [comment]
@CodiumAI-Agent /improve

**Review by mrbin264 — APPROVED** [review]

**Pull request merged**